pages/contribute/working_with_git.md

```diff
@@ -77,6 +77,7 @@ If you have never installed or run a Jekyll site locally on your computer, follo
 
     ```
     gem install jekyll bundler github-pages
+    bundle install
```

Comment:
think only one `gem install` or `bundle install` is necessary

Reply:
I get this error message:
Could not find (list of stuff) in locally installed gems
Run `bundle install` to install missing gems.

Reply:
yes but if you only run `bundle install` without `gem install`?

Reply:
And bundle install without gem install gives "Could not locate Gemfile"

Reply:
Hmm, that's odd would have thought [rdmkit](https://github.com/elixir-europe/rdmkit)/Gemfile does the trick

Reply:
I think we could take away 'github-pages' since with the bundle isntall it is taken care off

Reply:
@korbinib bundle install create the gem.lock file, which is not in the repo anymore since at this moment we are locked to the dependencies of GitHub Pages